syntax = "proto3";

package grpc_service.utility;

// Message for premise information
message PremiseInfo {
    string latitude = 1;
    string longitude = 2;
    string building_name = 3;
    string building_number = 4;
    string plot_number = 5;
    string address = 6;
    string district = 7;
    string street = 8;
    string city = 9;
    string plot = 10;
    string slug = 11;
    string premise_type = 12;
    string emirate = 13;
}
